packages/common/src/genesisStates/index.ts

```diff
@@ -7,12 +7,14 @@ const genesisStates: genesisStatesType = {
     '4': 'rinkeby',
     '42': 'kovan',
     '5': 'goerli',
+    '34180983699157880': 'yolov3',
```

Comment:
Same here, these should be chain ids right?

Reply:
Yes.